spec/javascripts/tests/WordCount_spec.js

```diff
@@ -0,0 +1,121 @@
+describe.only('WordCount', function(eventsWithPromises) {
```

Comment:
Remove the only function. I need to add the only keyword on the pre-commit for javascript files.